### PR TITLE
Create 2024-07-03.md

### PR DESCRIPTION
**Pull request by maximilien**
Initial minutes for 07/03 meeting

### CODE REVIEW

**Comment by maximilien** [comment]
Please peruse and suggest changes or approve

cc: @brian-jarvis-aws @planetf1 @hartm @Naomi-Wash @ryjones

File: meeting-minutes/2024/2024-07-03.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: 2024-06-05 TAC Meeting Record
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
title: 2024-07-03 TAC Meeting Record
```

File: meeting-minutes/2024/2024-07-03.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+No new items added
+
+## CBOM tooling presentation by Nicklas Körtge
+
+Nicklas Körtge (IBM Research - Zurich) presented his tooling that helps faciliate [CBOM](https://github.com/IBM/CBOM) creation and maintenance.
+
+His presentation covered:
+
+1. Show the OSS (Apache 2.0) [Sonor Cryptography Plugin](https://github.com/IBM/sonar-cryptography) github repository
+3. Demonstation of the live service that uses the plugin
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
3. Demonstration of the live service that uses the plugin
```

File: meeting-minutes/2024/2024-07-03.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+   * [Live service](https://cbomdb-frontend-scanservice.openshift-cluster-d465a2b8669424cc1f37658bec09acda-0000.eu-de.containers.appdomain.cloud) aviable to all to test and generate downloadable CBOM (Java and Python for now)
```
Comment (brian-jarvis-aws) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
   * [Live service](https://cbomdb-frontend-scanservice.openshift-cluster-d465a2b8669424cc1f37658bec09acda-0000.eu-de.containers.appdomain.cloud) available to all to test and generate downloadable CBOM (Java and Python for now)
```

**Review by brian-jarvis-aws — CHANGES_REQUESTED** [review]
Left a couple nitpicky spelling corrections, but the blocking item is to correct the date (line 3).

**Review by maximilien — COMMENTED** [review]
/lgtm

**Comment by maximilien** [comment]
Accepted your changes @brian-jarvis-aws

**Review by brian-jarvis-aws — APPROVED** [review]

File: meeting-minutes/2024/2024-07-03.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+## TAC members
+
+* [ ] Norman Ashley, Cisco
+* [x] Michael (Max)imilien, IBM
+* [x] Sam Stanwyck, NVIDIA
+* [ ] Sophie Schmieg, Google
+* [x] Brian Jarvis, Amazon Web Services Inc.
+* [ ] Thomas Bailleux, SandboxAQ
+
```
Comment (planetf1) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
* [ ] Nigel Jones, IBM
```

**Review by planetf1 — CHANGES_REQUESTED** [review]

**Comment by maximilien** [comment]
@planetf1 can you review please to unblock merge. Thx

File: meeting-minutes/2024/2024-07-03.md (hunk 4 above)
Comment (maximilien) [reply meeting-minutes/2024/2024-07-03.md]: Not sure why this is blocking merge

File: meeting-minutes/2024/2024-07-03.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+## TAC members
+
+* [ ] Norman Ashley, Cisco
+* [x] Michael (Max)imilien, IBM
+* [x] Sam Stanwyck, NVIDIA
+* [ ] Sophie Schmieg, Google
+* [x] Brian Jarvis, Amazon Web Services Inc.
+* [ ] Thomas Bailleux, SandboxAQ
+* [ ] Nigel Jones, IBM
```
Comment (planetf1) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
* [x] Nigel Jones, IBM
```

I was on the call

File: meeting-minutes/2024/2024-07-03.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+## Additional attendees
+* Yarkin Doroz (NVDIA)
+* Bryan Uhri (KeyFactor)
+* Alex Bozarth (IBM)
+* Nigel Jones (IBM)
```
Comment (planetf1) [inline meeting-minutes/2024/2024-07-03.md]:
```suggestion
```
removed from this section

**Review by planetf1 — CHANGES_REQUESTED** [review]
Updated the attendance. Will watch out for change then add approval as requested.

**Comment by ryjones** [comment]
@planetf1 check the latest commit

**Review by planetf1 — APPROVED** [review]
thanks for updating

**Pull request merged**